packages/kbn-reporting/generate_csv/impl/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@kbn/reporting-generate-csv",
```

Comment:
I'd prefer to not have this package namespaced under`reporting`. Let's just call it `@kbn/generate-csv` and have this code live in `packages/kbn-generate-csv`. LMK if there is a reason to keep it here.

Reply:
I changed the name in commit [81239f4](https://github.com/elastic/kibana/pull/151801/commits/81239f407b6467b400e4ad5e47466552874a7b10). Should it be outside of the kbn/reporting directory entirely or alongside kbn/reporting-common? I can see a case made for both so I kept it under the kbn-reporting directory but easy fix if it should be moved outside that directory, thanks!

Reply:
Disregard - I see your later comment thanks!